setup.json

```
@@ -48,8 +48,8 @@
         "pyyaml~=5.1.2",
         "reentry~=1.3",
         "simplejson~=3.16",
-        "sqlalchemy-utils~=0.36.0",
-        "sqlalchemy~=1.3.10",
+        "sqlalchemy-utils~=0.37.0",
```

```suggestion
        "sqlalchemy-utils==0.37.0",
```
See previous comment.